lib/msal-angular/docs/v2-docs/known-issues.md

```diff
@@ -1,5 +1,8 @@
 # Known issues for MSAL Angular v2
```

Comment:
Do we have these issues tracked as items in ADO to resolve?

Reply:
A few already are, I will update ADO with the rest.